src/modules/vendor/components/programs/cards/templates/QuestionAnswer.vue

```diff
@@ -56,34 +56,16 @@ export default {
       immediate: true,
     },
   },
+  async created () {
+    this.card.isQuestionAnswerMultipleChoiced = !!this.card.isQuestionAnswerMultipleChoiced;
```

Comment:
Pourquoi est-ce qu'il y en a besoin ?

Reply:
c'est par rapport au commentaire de Corentin pour initialiser la checkbox à `false` plutot que `undefined`

Reply:
ou alternative plus explicite :
`if (this.card.isQuestionAnswerMultipleChoiced === undefined) this.card.isQuestionAnswerMultipleChoiced = false`

Reply:
Que pensez-vous de mettre isQuestionAnswerMultipleChoiced par defaut a false dans le back ?

Reply:
ok on fait ca

Reply:
done